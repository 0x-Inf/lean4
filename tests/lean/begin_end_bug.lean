example (a b c : nat) : a = b → a = c → b = c :=
assume hab hac,
have b = a, begin trace a, symmetry, assumption end,
begin
  transitivity,
  exact this,
  exact hac
end
