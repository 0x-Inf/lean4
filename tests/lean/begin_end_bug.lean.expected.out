begin_end_bug.lean:3:24: error: unknown identifier 'a'
begin_end_bug.lean:8:0: error: invalid 'end', there is no open namespace/section
